src/components/org/PositionCard/components/PositionInstance.tsx

```diff
@@ -21,6 +23,8 @@ type PositionInstanceProps = {
 const PositionInstanceComponent: React.FC<PositionInstanceProps> = ({
     position,
     instance,
+    isFuture,
```

Comment:
Do we need these props in the Instance component?

Reply:
Removed